src/App.jsx

```diff
@@ -13,34 +15,103 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+const RESET = '';
+const SORT_ALPHABETICALLY = 'alphabet';
+const SORT_BY_LENGTH = 'length';
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function getPreparedGoods(goods, { sortField, query }) {
+  let preparedGoods = [...goods];
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  if (query) {
+    preparedGoods = preparedGoods.reverse();
+  }
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  preparedGoods.sort((good1, good2) => {
+    switch (sortField) {
+      case SORT_ALPHABETICALLY:
+        return good1.localeCompare(good2);
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+      case SORT_BY_LENGTH:
```

Comment:
*optional. Since `enum` isn't required
```suggestion
      case SortBy.LENGTH:
```